src/TerminalObject/Helper/StringLength.php

```diff
@@ -47,7 +47,7 @@ protected function withoutTags($str)
     {
         $this->setIgnoreTags();
 
-        return str_replace($this->ignore_tags, '', $str);
+        return str_replace($this->ignore_tags, '', (string)$str);
```

Comment:
This parameter is already declared as a string, either a parameter type should be added, or the code that is passing something other than a string here should be fixed, or both!

Reply:
This string cast needed here. Otherwise null values in some helpers throw deprecation errors

Reply:
Which helpers? It sounds like they should be fixed so they don't pass null when this method requires a string

Reply:
Test ColumnTest::it_can_output_an_uneven_array_of_arrays_as_columns() has column with null values
TerminalObject/Basic/Columns->getColumnWidth() throw deprecation if column has null values

You can simply run tests from PHP 8.1 to see all this fixed deprecations without BC brake